src/index.js

```diff
@@ -135,7 +137,7 @@ const { WORKSPACE_EXTERNAL } = SLACK_TARGETS;
 
 export const main = wrap(run)
   .with(dataAccess)
-  .with(auth)
+  .with(auth, { authHandlers: [LegacyApiKeyHandler, AdobeImsHandler] })
```

Comment:
providing a list of auth handler here seems a bit redundant. Do we have any case where we have a handler implemented, but not provided here?

maybe we can have controller specific handlers instead? Controllers can define which handlers they use?

Reply:
yes, we can do that, but not in step one please.

Reply:
will be expecting that in the step two then 🧖‍♂️